src/serialization/tokenizer.cpp

```diff
@@ -78,8 +78,8 @@ const token &tokenizer::next_token()
 		token_.type = token::END;
 		break;
 
-	case '<':
-		if (peek_char() != '<') {
+	case token::LESS_THAN:
```

Comment:
Isn't this an "angle bracket" rather than a "less than"?

Reply:
I'll change the name. I don't know that it currently supports checking multiple characters directly though - it only gets a single character at a time from the `istream`.